examples/create_hdr_encoding_with_master_display_information_hls.go

```diff
@@ -0,0 +1,317 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/bitmovin/bitmovin-go/bitmovin"
+	"github.com/bitmovin/bitmovin-go/bitmovintypes"
+	"github.com/bitmovin/bitmovin-go/models"
+	"github.com/bitmovin/bitmovin-go/services"
+)
+
+type H265CodecConfigDefinition struct {
+	width    *int64
+	height   *int64
+	crf      *float64
+	streamId *string
+	muxingId *string
+}
+
+const (
+	apiKey             = "YOUR API KEY"
+	inputHost          = "YOUR_HTTP_INPUT_HOST"
+	inputFilePath      = "/path/to/your/input/file.mov"
+	s3OutputAccessKey  = "YOUR_ACCESS_KEY"
+	s3OutputSecretKey  = "YOUR_SECRET_KEY"
+	s3OutputBucketName = "YOUR_BUCKET_NAME"
+	outputBasePath     = "golang_hls_example"
+)
```

Comment:
Golang certainly prefers camelCase for everything, including constants, but it's the one place where I disagree with the style guide.  I'm fine with keeping it this way though.